src/compiler/scala/tools/nsc/GenericRunnerSettings.scala

```diff
@@ -45,9 +46,10 @@ class GenericRunnerSettings(error: String => Unit) extends Settings(error) {
 
   val nc = BooleanSetting(
       "-nc",
-      "do not use the fsc compilation daemon") withAbbreviation "-nocompdaemon" withPostSetHook((x: BooleanSetting) => {_useCompDaemon = !x.value })
+      "do not use the fsc compilation daemon and shut it down if it is running").withAbbreviation("-nocompdaemon")
+      //.withDeprecationMessage("scripts use cold compilation by default; use -howtorun:fsc to start the compilation daemon")
```

Comment:
I'm in favor of adding the deprecation warning, so we can remove `-nc` eventually

Reply:
I'll need a better story for replacing `fsc -shutdown`. Besides `-howtorun:shutdown`. Not really sure anyone ever, anywhere, has ever used `-howtorun`; I think I tweaked it once. Maybe it should be renamed to `-whattorun`.